dumpling-overview.md

```diff
@@ -379,7 +377,7 @@ When Dumpling is exporting a large single table from TiDB, Out of Memory (OOM) m
 + Reduce the value of `--tidb-mem-quota-query` to `8589934592` (8 GB) or lower. `--tidb-mem-quota-query` controls the memory usage of a single query statement in TiDB.
 + Adjust the `--params "tidb_distsql_scan_concurrency=5"` parameter. [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) is a session variable which controls the concurrency of the scan operations in TiDB.
 
-### TiDB GC settings when exporting a large volume of data (more than 1 TB)
+### Set TiDB GC when exporting a large volume of data (more than 1 TB)
 
 When exporting data from TiDB (more than 1 TB), if the TiDB version is later than or equal to v4.0.0 and Dumpling can access the PD address of the TiDB cluster, Dumpling automatically extends the GC time without affecting the original cluster.
 
```

Comment:
Line 400~410 were removed from the [Chinese version](https://github.com/pingcap/docs-cn/pull/7143/files#diff-d167a2ebb86d802c7a7279da3de5de87e8b616ed0a27755ef0027a8db570c147L332). Please also delete them from English.

Reply:
OK done